example/advanced.py

```diff
@@ -58,24 +58,38 @@ def make_request(payload):
 
 
 def list_embedded():
-    from dotty_dict import dotty_l
+    from dotty_dict import dotty
 
-    # use dotty_l if you need support for lists
+    # dotty supports embedded lists
     # WARNING!
-    # Right now you can read and delete from multidimensional lists
-    # but setting value is supported only for one dimensional list.
+    # Dotty used to support lists only with dotty_l.
+    # This feature is depreciated - now lists have native support.
 
-    dot = dotty_l({
+    dot = dotty({
         'annotations': [
             {'label': 'app', 'value': 'webapi'},
             {'label': 'role', 'value': 'admin'},
-        ]
+        ],
+        'spec': {
+            'containers': [
```

Comment:
IMO it will be more valuable to check if we can access lists in lists.
Mixed dict and list works right now without any changes. This PR provides a way to access lists in lists and this fact should be more emphasize.

The example can be changed to:

```python
'containers': [
    ['gpu', 'tensorflow', 'ML'],
    ['cpu', 'webserver', 'sql'],
]
```

And then assertions :

```python
assert dot['spec.containers.0.0'] == 'gpu'
assert dot['spec.containers.0.1'] == 'tensorflow'
assert dot['spec.containers.0.2'] == 'ML'
assert dot['spec.containers.1.0'] == 'cpu'
assert dot['spec.containers.1.1'] == 'webserver'
assert dot['spec.containers.1.2'] == 'sql'
```

Reply:
Resolved